stop-dev.sh

```diff
@@ -11,10 +11,10 @@ ARGS="$@"
 # If no nodes defined, stop all.
 NODES="${ARGS:-1 2 3 4 5}"
 
-for i in 1 2 3 4 5; do
```

Comment:
This was hardcoded b/c there can only be 5 nodes with the current setup.

Reply:
You can still want to stop just one of the nodes for some reason, and it will stop all of them anyway, if you don't supply any arguments. So nothing lost, and something gained ;)

Reply:
In the current setup I don't think anything is gained. You can still only start up to 5 nodes (unless you modify the key generating script and/or add them to the local resources.

Reply:
That's the idea - you can add some keys to the local resources and start more nodes. Then the stopping script as it is now wouldn't handle them correctly.